Source/ui_macosx/ApplicationDelegate.mm

```diff
@@ -5,11 +5,13 @@
 #import "../../tools/PsfPlayer/Source/SH_OpenAL.h"
 #import "Globals.h"
 #import "PreferenceDefs.h"
+#include "ScreenShotUtils.h"
 #import "../ee/PS2OS.h"
 #import "../ScopedVmPauser.h"
 #import "../PS2VM_Preferences.h"
 #import "../AppConfig.h"
 
+
```

Comment:
There probably was an include that I didn't delete probably, I'll have this fixed.